apps/api/lib/validations/booking.ts

```diff
@@ -49,6 +49,15 @@ export const schemaBookingReadPublic = Booking.extend({
       locale: true,
     })
     .optional(),
+  payment: z
+    .array(
+      _PaymentModel.pick({
+        id: true,
+        success: true,
+        paymentOption: true,
+      })
+    )
+    .optional(),
```

Comment:
We currently pick the ID, success and paymentOption which are all quite nice to have fields for a quick state of the payments. For more details, the users can run a followup call to the `/payments/{id}` endpoint using the payment id returned here.